brainiak/utils/utils.py

```diff
@@ -105,6 +105,71 @@ def fast_inv(a):
         raise
 
 
+def sumexp_stable(data):
```

Comment:
We are using this for the objective function computation. This is only for logging.